action.yml

```diff
@@ -65,16 +66,21 @@ inputs:
     description: >-
       Directory that contains actions-runner software and scripts. E.g. /home/runner/actions-runner.
     required: false
+  runner-count:
+    description: >-
+      Number of instances to create.
+    required: false
+    default: '1'
 outputs:
   label:
     description: >-
       Name of the unique label assigned to the runner.
       The label is used in two cases:
       - to use as the input of 'runs-on' property for the following jobs;
       - to remove the runner from GitHub when it is not needed anymore.
-  ec2-instance-id:
+  ec2-instance-ids:
```

Comment:
If the name change here is considered breaking api-wise, we could always revert this and still treat it as one or more. I just thought it was more clear to make it plural.